recipe/bld.bat

```diff
@@ -0,0 +1,11 @@
+setlocal EnableDelayedExpansion
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST %LIBRARY_PREFIX%\etc\conda\%%F.d MKDIR %LIBRARY_PREFIX%\etc\conda\%%F.d
+    COPY %RECIPE_DIR%/%%F.bat %LIBRARY_PREFIX%\etc\conda\%%F.d\keras_%%F.bat
```

Comment:
@msarahan, thanks! Now the file seems to go in the right directory.  But the command to source the file on windows seems to be wrong `call "C:\Miniconda\Scripts\activate.bat" "C:\bld\keras_1487717779224\_t_env"  1>nul 2>&1`